deployment/terraform/coordinator.go

```diff
@@ -113,9 +115,34 @@ func (t *Terraform) StartCoordinator() error {
 	}
 
 	mlog.Info("Starting the coordinator")
-	cmd := "sudo service ltcoordinator start"
-	if out, err := sshc.RunCommand(cmd); err != nil {
-		return fmt.Errorf("error running ssh command: output: %q, error: %w", out, err)
+	data, err = json.Marshal(struct {
+		CoordinatorConfig coordinator.Config
+		LoadTestConfig    loadtest.Config
+	}{
+		*coordinatorConfig,
+		*agentConfig,
+	})
+	if err != nil {
+		return err
+	}
+	// TODO: create a wrapper to simplify coordinator's HTTP API usage.
+	id := fmt.Sprintf("%s-coordinator-%d", t.config.ClusterName, 0)
+	apiURL := fmt.Sprintf("http://%s:4000/coordinator/%s", ip, id)
```

Comment:
Should we also take the port from the config somewhere later? The port is actually exposed via a flag, it's just not captured in a config file.

Reply:
Yes, we can probably make a ticket for that but that might mean a new config file (for the API server).